### PR TITLE
Preparing for new plans

### PR DESCRIPTION
**Pull request by sbastn**
This PR:
- Adds a new page where we explain the new plans
- Converts the existing personal-vs-pro page to a more generic plan difference page: `articles/differences-between-dnsimple-plans/`
- Replace mentions of specific plan names by linking to a plan difference page

Fixes https://github.com/dnsimple/dnsimple-support/issues/1050

TODO:
- [x] Redirect where the user can see the differences between plans.
- [x] Review `/articles/understanding-invoice/#difference-between-unused-time-and-remaining-time`.
- [x] Review `/articles/dns-hosting/`.

### CODE REVIEW

File: content/index.html [hunk 1]
```diff
@@ -14,9 +14,9 @@ <h3>Getting started</h3>
   </div>
   <div class="tc card w-100 w-30-ns pa3 mb3 br2"">
     <i class="db f2 fa fa-user-check"></i>
-    <h3>Personal plan guide</h3>
-    <p>Make the most of your personal plan subscription.</p>
-    <a href="/articles/getting-started-personal-plan">Personal plan</a>
+    <h3>Solo plan guide</h3>
```
Comment (aeden) [inline content/index.html:17]: I know this was here before as "Personal plan guide" but I think I'd rather replace it with something that is not plan-specific, or remove it altogether.

File: categories/dnsimple.yaml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 Getting started:
 - "Getting Started"
-- "Getting Started With Your Personal Plan"
+- "Getting Started With Your Solo Plan"
```
Comment (aeden) [inline categories/dnsimple.yaml:3]: While this was here before for the Personal Plan, I think we should remove it unless we have a really good reason to keep it.

File: content/articles/dnsimple-plans.markdown [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+categories:
+- DNSimple
+---
+
+# DNSimple Plans
+
+## All plans include
+
+- Register, transfer, and renew domains. [See TLD prices](https://dnsimple.com/tlds)
+- $2 per month per authoritative DNS zone with Anycast DNS and multi-layer DDoS defense.
```
Comment (aeden) [inline content/articles/dnsimple-plans.markdown:13]:
```suggestion
- Primary or secondary DNS zones with Anycast DNS and multi-layer DDoS defense, billed monthly per zone.
```

File: content/articles/dnsimple-plans.markdown [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- $2 per month for domain email forwarding.
```
Comment (aeden) [inline content/articles/dnsimple-plans.markdown:14]:
```suggestion
- Email forwarding, billed monthly per domain.
```

File: content/articles/dnsimple-plans.markdown [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+Custom subscription fee.
+
+Activates the following features:
+- Priority support
+- Single sign-on with Okta or Google Workspace
+- Vanity Name Servers.
+- High-volume zones.
+- Discounted domain registration, transfer, and renewal fees.
+- DNS zone discounts.
+- 100% SLA.
```
Comment (aeden) [inline content/articles/dnsimple-plans.markdown:49]:
```suggestion
- 100% SLA available.
```

File: content/articles/dnsimple-plans.markdown [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+## Enterprise
+
+Custom subscription fee.
+
+Activates the following features:
+- Priority support
+- Single sign-on with Okta or Google Workspace
+- Vanity Name Servers.
+- High-volume zones.
+- Discounted domain registration, transfer, and renewal fees.
```
Comment (aeden) [inline content/articles/dnsimple-plans.markdown:47]:
```suggestion
- Discounted domain registration, transfer, and renewal fees for high-volume TLDs.
```

File: content/articles/email-forwarding.markdown [hunk 7]
```diff
@@ -14,23 +14,9 @@ categories:
 
 ---
 
-Email forwarding is enabled and billed on a per-domain basis. The cost is $2 per-domain per-month, which totals $24 per-domain per-year on a yearly subscription plan.
+Email forwarding is enabled and billed on a per-domain basis. The cost is $2 per-domain per-month.
```
Comment (aeden) [inline content/articles/email-forwarding.markdown:17]:
```suggestion
Email forwarding is enabled and billed monthly on a per-domain basis.
```

File: content/articles/dnsimple-plans.markdown [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+## Solo
+
+No base subscription fee.
+
+DNSimple's Solo plan provides anyone with a domain management platform where they can manage DNS, domains, and certificates with multiple providers through a single control plane.
+
+## Teams
+
+$199 per month base subscription fee.
```
Comment (aeden) [inline content/articles/dnsimple-plans.markdown:24]:
```suggestion
Requires base monthly subscription fee.
```

File: content/articles/dnsimple-plans.markdown [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+Custom subscription fee.
+
+Activates the following features:
+- Priority support
+- Single sign-on with Okta or Google Workspace
+- Vanity Name Servers.
+- High-volume zones.
+- Discounted domain registration, transfer, and renewal fees.
+- DNS zone discounts.
```
Comment (aeden) [inline content/articles/dnsimple-plans.markdown:48]:
```suggestion
- DNS zone discounts available.
```

**Review by aeden — CHANGES_REQUESTED** [review]
First pass review to remove static pricing. We will need to go through it again.

File: content/index.html (hunk 1 above)
Comment (sbastn) [reply content/index.html:17]: I'll remove it for now.

**Comment by sbastn** [comment]
@itsalyse can you please have a pass at this? Let me know if you have any questions!

File: content/articles/dns-hosting.markdown [hunk 10]
```diff
@@ -85,26 +85,15 @@ Each new domain added to an account has DNS hosting enabled by default, whether
...
-- Personal plan ($0.50 per month)
-- Professional plan ($2.00 per month)
-- Enterprise plan (custom pricing)
+If the domain is registered with us but delegated to another DNS provider, deactivating the DNS zone won't change the delegation. The domain will continue to resolve with the other DNS provider.
 
 
 ## Use cases
 
-There are a few cases where you may want to turn off DNS hosting for a domain:
+There are a few cases where you may want to deactivating a zone:
```
Comment (aeden) [inline content/articles/dns-hosting.markdown:95]:
```suggestion
There are a few cases where you may want to deactivate a zone:
```

File: content/articles/dnsimple-plans.markdown [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+
+## Solo
+
+No base subscription fee.
+
+DNSimple's Solo plan provides anyone with a domain management platform where they can manage DNS, domains, and certificates with multiple providers through a single control plane.
+
+## Teams
+
+Requires base monthly subscription fee.
```
Comment (aeden) [inline content/articles/dnsimple-plans.markdown:26]:
```suggestion
Requires a base monthly subscription fee.
```

File: content/articles/dnsimple-plans.markdown [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+
+Activates the following features:
+- HTTPS URL redirects.
+- Let's Encrypt custom hostname certificates.
+- Wildcard certificates.
```
Comment (aeden) [inline content/articles/dnsimple-plans.markdown:31]:
```suggestion
- Let's Encrypt wildcard certificates.
```

File: content/articles/dnsimple-services.markdown [hunk 13]
```diff
@@ -11,7 +11,7 @@ DNSimple provides essential services for every Internet-connected system: domain
 
 ## DNS hosting
 
-We bill DNS hosting as a monthly subscription. If you sign up for, or upgrade to, a yearly plan, you may receive a discount on your total subscription price. Pricing is available on our [pricing page](https://dnsimple.com/pricing).
+We bill DNS hosting as on top of your subscription base fee. Pricing is available on our [pricing page](https://dnsimple.com/pricing).
```
Comment (aeden) [inline content/articles/dnsimple-services.markdown:14]:
```suggestion
Each DNS zone is billed separately from any base subscription fee. Pricing is available on our [pricing page](https://dnsimple.com/pricing).
```

File: content/articles/slack-integration.markdown [hunk 14]
```diff
@@ -14,7 +14,7 @@ categories:
 
 ---
 
-The Slack app allows DNSimple customers to register and manage domains from their team's Slack Channel. The app is available for Professional, Enterprise and Reseller plans.
+The Slack app allows DNSimple customers to register and manage domains from their team's Slack Channel. The app is available  on certain plans. [See the difference between plans](/articles/dnsimple-plans/).
```
Comment (aeden) [inline content/articles/slack-integration.markdown:17]:
```suggestion
The Slack app allows DNSimple customers to register and manage domains from their team's Slack Channel. The app is available on certain plans. [See the difference between plans](/articles/dnsimple-plans/).
```

File: content/articles/subscription-renewals.markdown [hunk 15]
```diff
@@ -14,7 +14,7 @@ categories:
 
 ---
 
-Depending on the billing cycle for your plan, your subscription will be renewed either monthly or yearly. The payment for the renewal will be charged to the card on file or, if you have credit, deducted from from the credit balance.
+Your subscription will be renewed every month. The payment for the renewal will be charged to the card on file or, if you have credit, deducted from from the credit balance.
```
Comment (aeden) [inline content/articles/subscription-renewals.markdown:17]:
```suggestion
Your subscription will be renewed each month. The payment for the renewal will be charged to the card on file or, if you have credit, deducted from from the credit balance.
```

**Review by aeden — CHANGES_REQUESTED** [review]
Few changes to make.

**Review by itsalyse — APPROVED** [review]

**Review by aeden — APPROVED** [review]

**Pull request merged**